nixos/doc/manual/configuration/gpu-accel.xml

```diff
@@ -100,7 +100,7 @@ Platform Vendor      Advanced Micro Devices, Inc.</screen>
        support. For example, for Gen8 and later GPUs, the following
        configuration can be used:
 
-	      <programlisting><xref linkend="opt-hardware.opengl.extraPackages"/> = [
+	      <programlisting><xref linkend="opt-hardware.opengl.extraPackages"/> = with pkgs; [
   intel-compute-runtime
```

Comment:
with pkgs should probably be avoid if possible, for an example, we could probably do
```suggestion
	      <programlisting><xref linkend="opt-hardware.opengl.extraPackages"/> = [
  pkgs.intel-compute-runtime
```